migrations/003.do.top-cids.sql

```diff
@@ -0,0 +1,214 @@
+-- Generated using scripts/top-cids.js
+INSERT INTO retrieval_templates (cid, provider_address, protocol) VALUES ('bafybeieijah7ifacactkvsmxebxgfl4kyvoa3al6fyppwyohedb7vvyzl4', '/ip4/211.254.148.138/tcp/8180', 'graphsync');
+INSERT INTO retrieval_templates (cid, provider_address, protocol) VALUES ('bafybeiezeds576kygarlq672cnjtimbsrspx5b3tr3gct2lhqud6abjgiu', '/dns4/elastic.dag.house/tcp/443/wss', 'bitswap');
+INSERT INTO retrieval_templates (cid, provider_address, protocol) VALUES ('QmVBBwBaKNPixLtauB869E9RzqTycqTXVQUN6oN1XAtPZj', '/ip4/108.89.91.150/tcp/46717', 'graphsync');
```

Comment:
I thought about refactoring it to output that, but since I already had the multiple insert version and don't see an actual improvement (some bytes saved, minimal time saved?) I don't see a need to change it.